### PR TITLE
Add 3 required configs for FreedomChat, Fix typos.

### PR DESCRIPTION
**Pull request by Kichura**
This fixes a few typos in the entire server setup guide and introduces 3 new configs for FreedomChat itself.

### CODE REVIEW

File: en-us/server-setup.md [hunk 1]
```diff
@@ -18,33 +18,33 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
...
 - **Velocity**: set `force-key-authentication` to `false` in _velocity.toml_
   - Same comments apply as for "all servers" above.
 - **BungeeCord/Waterfall**: set `enforce_secure_profile` to `false` in _config.yml_
   - Same comments apply as for "all servers" above.
-- **Paper/Pufferfish/Purpur/Spigot**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin - no config required
-- **Fabric/Quilt/Forge**: install _one_ of the following plugins:
-  - [NoChatReport](https://modrinth.com/mod/no-chat-report) - compatible with most chat mods/vanilla chat, no config required 
-  - [No Chat Reports](https://www.curseforge.com/minecraft/mc-mods/no-chat-reports) - same mod as in FO; FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat
+- **Spigot/Paper/Pufferfish/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin and ensure `rewrite-chat`, `claim-secure-chat-enforced` and `send-prevents-chat-reports-to-client` are all set to `true` inside the FreedomChat's config folder for best effects.
+- **Forge/Fabric/Quilt**: install _one_ of the following plugins:
```
Comment (Madis0) [inline en-us/server-setup.md:40]: no
fabric first

File: en-us/server-setup.md [hunk 2]
```diff
@@ -18,33 +18,33 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
...
 - **Velocity**: set `force-key-authentication` to `false` in _velocity.toml_
   - Same comments apply as for "all servers" above.
 - **BungeeCord/Waterfall**: set `enforce_secure_profile` to `false` in _config.yml_
   - Same comments apply as for "all servers" above.
-- **Paper/Pufferfish/Purpur/Spigot**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin - no config required
-- **Fabric/Quilt/Forge**: install _one_ of the following plugins:
-  - [NoChatReport](https://modrinth.com/mod/no-chat-report) - compatible with most chat mods/vanilla chat, no config required 
-  - [No Chat Reports](https://www.curseforge.com/minecraft/mc-mods/no-chat-reports) - same mod as in FO; FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat
+- **Spigot/Paper/Pufferfish/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin and ensure `rewrite-chat`, `claim-secure-chat-enforced` and `send-prevents-chat-reports-to-client` are all set to `true` inside the FreedomChat's config folder for best effects.
```
Comment (Madis0) [inline en-us/server-setup.md:39]: paper first
also can you link me the config wiki, last I checked there was no config at all

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: en-us/server-setup.md (hunk 2 above)
Comment (Kichura) [reply en-us/server-setup.md:39]: Same as above, Spigot has existed first before paper, pufferfish and purpur respectively aswell.

File: en-us/server-setup.md (hunk 1 above)
Comment (Kichura) [reply en-us/server-setup.md:40]: I sorted this in order as forge existed first before fabric and quilt did respectively.

Comment (Madis0) [reply en-us/server-setup.md:40]: your logic is correct, I'm just focused on what FO users are more likely to use/be in 😆

Comment (osfanbuff63) [reply en-us/server-setup.md:40]: You could also sort them alphabetically, as in `Fabric/Forge/Quilt`, but honestly I think the current one is fine

File: en-us/server-setup.md (hunk 2 above)
Comment (Madis0) [inline en-us/server-setup.md:39]:
```suggestion
- **Spigot/Paper/Pufferfish/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin and ensure `rewrite-chat`, `claim-secure-chat-enforced` and `send-prevents-chat-reports-to-client` are all set to `true` inside the FreedomChat's config folder. FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat.
```

Comment (osfanbuff63) [reply en-us/server-setup.md:39]: Also, `the FreedomChat's config folder`?

File: en-us/server-setup.md [hunk 3]
```diff
@@ -18,33 +18,33 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
...
 - **Velocity**: set `force-key-authentication` to `false` in _velocity.toml_
   - Same comments apply as for "all servers" above.
 - **BungeeCord/Waterfall**: set `enforce_secure_profile` to `false` in _config.yml_
   - Same comments apply as for "all servers" above.
-- **Paper/Pufferfish/Purpur/Spigot**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin - no config required
-- **Fabric/Quilt/Forge**: install _one_ of the following plugins:
-  - [NoChatReport](https://modrinth.com/mod/no-chat-report) - compatible with most chat mods/vanilla chat, no config required 
-  - [No Chat Reports](https://www.curseforge.com/minecraft/mc-mods/no-chat-reports) - same mod as in FO; FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat
+- **Spigot/Paper/Pufferfish/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin and ensure `rewrite-chat`, `claim-secure-chat-enforced` and `send-prevents-chat-reports-to-client` are all set to `true` inside the FreedomChat's config folder. FO users will get a ![green ✅](https://i.ibb.co/LPXNKRM/green.png) icon near chat.
```
Comment (Madis0) [inline en-us/server-setup.md:39]: Why did spigot get to be first again

Comment (Kichura) [reply en-us/server-setup.md:39]: > Why did spigot get to be first again

https://github.com/Fabulously-Optimized/wiki/pull/49#discussion_r999743713

Comment (osfanbuff63) [reply en-us/server-setup.md:39]: > Same as above, Spigot has existed first before paper, pufferfish and purpur respectively aswell.
>
> As for the freedomchat config's wiki, [right here.](https://github.com/e-im/FreedomChat/blob/main/src/main/resources/config.yml)

**Pull request merged**